dashboard/app/models/script_level.rb

```diff
@@ -67,6 +67,16 @@ def end_of_stage?
     stage.script_levels.to_a.last == self
   end
 
+  def long_assessment?
+    if self[:assessment]
```

Comment:
you can just use `assessment` here. It is not common to use the hash form to access ActiveRecord attributes.